src/ReactWrapper.jsx

```diff
@@ -243,11 +277,14 @@ class ReactWrapper {
    * @param {Function} cb - callback function
    * @returns {ReactWrapper}
    */
-  setProps(props, callback = undefined) {
+  setProps(props, callback = noop) {
     if (this.root !== this) {
       throw new Error('ReactWrapper::setProps() can only be called on the root');
     }
-    this.component.setChildProps(props, callback);
+    this.component.setChildProps(props, () => {
+      this.update();
+      callback();
```

Comment:
no it should default to a noop because the previous behavior was that it wasn't required.